tests/MongoDB.Driver.Tests/JsonDrivenTests/JsonDrivenRunCommandTest.cs

```diff
@@ -92,6 +100,12 @@ protected override void SetArgument(string name, BsonValue value)
                 case "command":
                     _command = value.AsBsonDocument;
                     return;
+                
+                case "databaseOptions":
+                    _readConcern = value.AsBsonDocument.Contains("readConcern")
+                        ? ReadConcern.FromBsonDocument(value["readConcern"].AsBsonDocument)
+                        : _readConcern;
+                    return;
```

Comment:
I think this use of ternary is crazy.

I'd rather see something like:

    if (value.AsBsonDocument.TryGetValue("readConcern", out BsonValue readConcernValue)
    {
        _readConcern = ReadConcern.FromBsonDocument(readConcernValue.AsBsonDocument);
    }
    return;

Reply:
Okay, I've taken your changes with a minor tweak!

Reply:
Oh nice... I didn't know you could use var like that.

Reply:
😺